tests/src/FunctionalJavascript/WebformCivicrmTestBase.php

```diff
@@ -19,6 +19,7 @@ abstract class WebformCivicrmTestBase extends CiviCrmTestBase {
     'webform_ui',
     'webform_civicrm',
     'token',
+    'CKEditor',
```

Comment:
Dont think we need this. We created a helper function to fill value in ckeditor. See - https://github.com/colemanw/webform_civicrm/blob/6.x/tests/src/FunctionalJavascript/WebformCivicrmTestBase.php#L663

So maybe, 

    $this->fillCKEditor('civicrm_1_activity_1_activity_details[value]', $params['body']);

should be able to fill values on the webform? @KarinG

Reply:
If I look at the screenshots - the field Activity Details is not on the View of the Webform (with this type change). But on my 2 live sites with this PR it is. Are we missing a Webform submodule in the test?

Reply:
ah ok - i see you just need to replace `CKEditor` with `ckeditor` the fix the error @KarinG. Can you push that update pls?

Reply:
Ah yes l - will do in a few min - just getting set up in the Aquaplex at the Sport Centre

Reply:
Ok running that on my local now (w/ '`ckeditor`')

Reply:
Passed on my local test env -> pushing now. I think that was it! Thank you for spotting that 👍